### ISSUE TITLE
Can it use the model trained from darknet?

### ISSUE THREAD

**Issue by Griovwe** [experiencor/keras-yolo3#16]
Hi,
Thank you for the sharing.
I am wondering if I can use the model trained from darknet?
And is the available pre-trained model trained by raccoon-dataset?

Thank you

**Comment by experiencor** [comment]
Yes, you can convert the model from darknet to keras using yolo3_one_file_to_detect_them_all.py. You need to change the number of classes though.

I will release many pretrained models in the next few days. Check out later.
